packages/core2/src/constants/index.ts

```diff
@@ -0,0 +1,12 @@
+export { ApplicationMode } from "./ApplicationMode";
+
+export { SLICE_MACHINE_USER_AGENT, APIEndpoints, APITokens } from "./api";
+
+export {
+	SLICE_MACHINE_NPM_PACKAGE_NAME,
+	SLICE_MACHINE_CONFIG_JS,
+	SLICE_MACHINE_CONFIG_TS,
+	SLICE_MACHINE_CONFIG_FILENAMES,
+	TS_CONFIG_FILENAME,
+	DEFAULT_SLICE_SCREENSHOT_URL,
+} from "./project";
```

Comment:
Rather than group constants by category (it's too subjective and could lead to strange organization), we can:

- Create one file per constant, removing categories altogether. If constants are very similar (e.g. `SLICE_MACHINE_CONFIG_JS` and `_TS`), that could be a single obj-as-const enum.
- Retain a single `constants.ts` file. It's simple. It may get unwieldy eventually.

Any other ideas? I feel the categorization here (`ApplicationMode.ts`, `api.ts`, `project.ts`) is unnecessary and adds overhead.

Reply:
Barrel files (i.e. `index.ts`) can cause messiness in other files. Autocomplete might import constants from the individual source files (e.g. `constants/api.ts`), while importing from the barrel file in other times.

In my opinion, it's cleaner to remove `index.ts` and import from the source files directly. That ensures there's only one way to import code.

Reply:
Let's go back to individual file